.pre-commit-config.yaml

```diff
@@ -57,6 +57,7 @@ repos:
 
         language: system
         pass_filenames: false
+        files: '^timemachine/cpp/src/'
```

Comment:
Actually is there a reason this doesn't only look at the `wrap_kernels.cpp`? Presumably the typing stubs are only for pybind?